### ISSUE TITLE
Non conforming files

### ISSUE THREAD

**Issue by clairey-zx81** [sdbs-uni-p/usage-of-not#2]
FYI: about 52 files are not JSON schemas.
For example:

- File `pp_69889.json` is not a JSON schema, but a JSON which contains a schema at some sublevel.
It should be deleted or the schemas should be extracted.

- File `pp_24618.json` contains a list of schemas.

- ...

**Comment by miniHive** [comment]
Yes, I can confirm the issue. We validated all files against a JSON validator, but a small number of files are not schemas (but nevertheless valid JSON).
